.github/ci-heapster.sh

```diff
@@ -105,6 +105,15 @@ install_yices() {
   rm -rf "yices$ext" "yices-$YICES_VERSION"
 }
 
+install_yasm() {
+  is_exe "$BIN" "yasm" && return
+  if [[ "$RUNNER_OS" = "Linux" ]]; then
+    sudo apt-get update -q && sudo apt-get install -y yasm
+  else
+    brew install yasm
+  fi
+}
+
```

Comment:
This makes me think the above may be undoing a fix, since we no longer need YASM. I also notice that this PR _moves_ this file, which would break the existing CI. Maybe it makes more sense to modify the workflow file to use `ci.sh` and get rid of the changes that created `ci-heapster.sh`?

Reply:
yes, I think you're right. I wasn't focused on how CI had been modified in heapster